src/interp_routines.F

```diff
@@ -554,10 +562,13 @@ subroutine vinterp_flx5(stag,ix,jy,kz,c1,c2,rrw,dumz,a)
         endif
       enddo
       enddo
+    ENDDO
 
-      !----
+    !----
 
-      k = 3
+    k = 3
+    !$acc parallel loop gang vector collapse(2) default(present)
```

Comment:
Do we need OpenMP directive here?

Reply:
Yes we do, I have added in OpenMP directives.